tools/SetupFlow/DevHome.SetupFlow/Views/AddRepoDialog.xaml

```diff
@@ -1,318 +1,457 @@
+    xmlns:i="using:Microsoft.Xaml.Interactivity"
+    xmlns:ic="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:models="using:DevHome.SetupFlow.Models"
+    x:Name="AddRepoContentDialog"
+    x:Uid="CloneRepoDialog"
+    CornerRadius="8"
+    DefaultButton="Primary"
+    IsPrimaryButtonEnabled="{x:Bind AddRepoViewModel.ShouldEnablePrimaryButton, Mode=OneWay}"
+    PrimaryButtonClick="AddRepoContentDialog_PrimaryButtonClick"
+    PrimaryButtonText="{x:Bind AddRepoViewModel.PrimaryButtonText, Mode=OneWay}"
+    Style="{StaticResource DefaultContentDialogStyle}"
+    mc:Ignorable="d">
     <ContentDialog.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
-                <ResourceDictionary Source="ms-appx:///DevHome.SetupFlow/Styles/SetupFlowStyles.xaml"/>
+                <ResourceDictionary Source="ms-appx:///DevHome.SetupFlow/Styles/SetupFlowStyles.xaml" />
                 <ResourceDictionary>
-                    <converters:BoolToVisibilityConverter x:Key="NegatedBoolToVisibilityConverter" TrueValue="Collapsed" FalseValue="Visible"/>
-                    <converters:BoolToVisibilityConverter x:Key="NegateBoolConverter" TrueValue="False" FalseValue="True"/>
-                    <converters:BoolToObjectConverter x:Key="BoolToGlyphConverter" TrueValue="&#xF0BD;" FalseValue="&#xF03F;"/>
-                    <converters:BoolNegationConverter x:Key="BoolToNegationConverter"/>
+                    <converters:BoolToVisibilityConverter
+                        x:Key="NegatedBoolToVisibilityConverter"
+                        FalseValue="Visible"
+                        TrueValue="Collapsed" />
+                    <converters:BoolToVisibilityConverter
+                        x:Key="NegateBoolConverter"
+                        FalseValue="True"
+                        TrueValue="False" />
+                    <converters:BoolToObjectConverter
+                        x:Key="BoolToGlyphConverter"
+                        FalseValue="&#xF03F;"
+                        TrueValue="&#xF0BD;" />
+                    <converters:BoolNegationConverter x:Key="BoolToNegationConverter" />
                 </ResourceDictionary>
             </ResourceDictionary.MergedDictionaries>
         </ResourceDictionary>
     </ContentDialog.Resources>
-    <!-- ContentDialog has a padding of 24 on left/right. Extend the ScrollViewer there to show the scroll bar at the edge of the dialog. -->
-    <ScrollViewer VerticalScrollMode="Auto" VerticalScrollBarVisibility="Auto" Padding="24,0" Margin="-24,0">
-        <StackPanel x:Name="AddRepoStackPanel" Orientation="Vertical" Spacing="5" MinWidth="450" MinHeight="550"  MaxWidth="450">
+    <!--  ContentDialog has a padding of 24 on left/right. Extend the ScrollViewer there to show the scroll bar at the edge of the dialog.  -->
+    <ScrollViewer
+        Margin="-24,0"

```

Comment:
It might be cleaner to remove the margin from the Scrollviewer by setting `<Thickness x:Key="ContentDialogPadding">0,0,0,0</Thickness>` in the Resource Dictionary. If you want get rid of the negative margin. (We do it in AddWidgetDialog)

Reply:
Tried that.  
![image](https://github.com/microsoft/devhome/assets/2517139/b17505af-f5f3-4e4a-8999-3a402632af18)
  Won't fix.